package.json

```diff
@@ -3,7 +3,7 @@
 	"version": "0.1.0",
 	"private": true,
 	"scripts": {
-		"dev": "API_URL=http://localhost:3000/api next dev",
+		"dev": "next dev",
```

Comment:
I think using cross-env here might do the trick: https://www.npmjs.com/package/cross-env

Reply:
I was also thinking about that, but sometimes these "cross"-env libraries can still not be cross platform, I chose to implement it by hardcoding the value.

Shall I implement it using the specified library?